site/pages/react/authenticate-users.mdx

```diff
@@ -86,6 +86,43 @@ export default function MyLoginPage() {
 }
 ```
 
+### Social Login
+
+You can allow uers to sign in with a Oauth provider like Google, Facebook, or Auth0. There are two login flows that are possible: the redirect flow goes to the auth provider's login page and the popup flow has the auth provider login screen to open as a popup.
```

Comment:
```suggestion
Allow users to sign in with an OAuth provider like Google, Facebook, or Auth0. This will trigger a popup window or a redirect to the auth provider's login page.
```

- write declaratively. "Allow users" rather than "you can allow"
- 'an' before OAuth
- OAuth has a capital A